pkgs/development/python-modules/djangorestframework-jwt/default.nix

```
@@ -0,0 +1,39 @@
+{ stdenv
+, fetchPypi
+, django
+, djangorestframework
+, buildPythonPackage
+, tox
+}:
+
+buildPythonPackage rec {
+  pname = "djangorestframework-jwt";
+  version = "1.11.0";
+  name = "${pname}-${version}";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "5efe33032f3a4518a300dc51a51c92145ad95fb6f4b272e5aa24701db67936a7";
+  };
+
+  buildInputs = [ django djangorestframework ];
+
+  # remove unused dependency, that causes tests to fail
+  preBuild = ''
+    sed -i "s/'PyJWT>=1.5.2,<2.0.0',//g" setup.py
+  '';
+
+  checkInputs = [ tox ];
+
+  doCheck = true;
+  checkPhase = ''
+    tox
```

```suggestion
    ./runtests.py
```
However: https://github.com/GetBlimp/django-rest-framework-jwt/pull/467